src/constants/colors/decimal-color-candy.ts

```diff
@@ -0,0 +1 @@
+export const DECIMAL_COLOR_CANDY = 15562905;
```

Comment:
Why not "NUMERICAL"? Because "DECIMAL" is either relative to 10 or with a comma/point value

Reply:
Agreed